config.go

```diff
@@ -722,7 +722,9 @@ func DefaultConfig() Config {
 		CoinSelectionStrategy:     defaultCoinSelectionStrategy,
 		KeepFailedPaymentAttempts: defaultKeepFailedPaymentAttempts,
 		RemoteSigner: &lncfg.RemoteSigner{
-			Timeout: lncfg.DefaultRemoteSignerRPCTimeout,
+			SignerRole:     lncfg.DefaultInboundWatchOnlyRole,
+			Timeout:        lncfg.DefaultRemoteSignerRPCTimeout,
```

Comment:
Playing around with this, I think we should have a separate "startup timeout" that we set to a default value of 5 minutes. That's how long the `watchonly-outbound` will want to wait. If we use the default 5 seconds, users are for sure going to run into this. Then we also don't need to instruct them to change a timeout in the tutorial doc.

Reply:
Good idea! Added a "startup timeout" :)